helm/aws-operator-chart/templates/03-configmap.yaml

```diff
@@ -28,6 +28,7 @@ data:
         trustedAdvisor:
           enabled: '{{ .Values.Installation.V1.Provider.AWS.TrustedAdvisor.Enabled }}'
         vaultAddress: '{{ .Values.Installation.V1.Auth.Vault.Address }}'
+        vpcPeerID: '{{ .Values.Installation.V1.Provider.AWS.VPCPeerID }}'
```

Comment:
Right now that comes from `kubernetesd` the same way. We drop the CR support for the rather static value of the CP and configure the operator directly.